ImagePicker/AsynchronousOperation.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AsynchronousOperation.swift
+//  ImagePicker
+//
+//  Created by Peter Stajger on 13/04/2018.
+//  Copyright © 2018 Inloop. All rights reserved.
+//
+
+import Foundation
+
+///
+/// Provides primitives for wrapping Operation with asynchronous code that can
+/// be enqueued in a OperationQueue and run serially.
+///
+class AsynchronousOperation : Foundation.Operation {
```

Comment:
We do have INLAyncOperation github repo, consider using it

Reply:
@radimhalfar I don't want to include any other dependencies to this project if it's not necessary.